language/tools/move-unit-test/src/extensions.rs

```diff
@@ -73,3 +94,26 @@ fn print_table_extension<W: Write>(w: &mut W, extensions: &mut NativeContextExte
 
 #[cfg(feature = "table-extension")]
 static DUMMY_RESOLVER: Lazy<BlankStorage> = Lazy::new(|| BlankStorage);
+
+
+#[cfg(test)]
+mod tests {
+    use better_any::{Tid, TidAble};
```

Comment:
I don't see this being used?

Reply:
Not sure what you mean. The test is executed. On needs to have both imports for `#[derive(Tid)]`.

Reply:
indeed... sometimes I wish I could delete comments, but at least I can resolve this one :)